### DISCUSSION TITLE
On Campus Job

### DISCUSSION THREAD

**Discussion by sai-prash** [snigi-gupta/AttendingUniversityAtBuffalo#6] [Q&A] [answered]
- Will a student be able to manage/balance 4 courses and a on campus job (10 -20 hours) in his 1st year as you have mentioned the study time required will be 40 hours every week?
- Will a on campus part time job be available for students?

**Comment by snigi-gupta** [comment, marked as answer]
@sp131 

To be honest I did not work part-time because I felt that I will not be able to manage both things properly. On the other hand, my close friend did work at CDS (Campus Dining Service) and also did well in his courses. It is very subjective to one's potential. Also depends on the courses you have and the workload.

I would suggest if you do have financial constraints, work at CDS for a month. See if you are able to manage or not and then quit incase it is too overwhelming.

If you want to join a part-time for fun, extra cash, then the above strategy works. But if you want to join because everyone else is doing the same then rethink once (people tend to do a lot of things coz the crowd is doing it too).

Some people join part-time for SSN only. I never understood the logic of getting an SSN in the first sem and how that is beneficial. You get an SSN when you have a job. And when you have an SSN then your credit score and etc. are tracked and calculated. Having an SSN from the first sem has no benefits than getting your SSN in your first job in my opinion. 

Part-time jobs at the campus:

- CDS - easy to get, most popular, your workload depends on what job you are given. Some jobs are lightweight, while others are not. Talk to the manager in advance and ask what duties can you be assigned in various shifts. This job will not help you with your resume. $11.80 per hour, usually.

- RA/TA/GA - difficult to get, workload can be easy to medium. Given by a professor. You can reach out to other department profs too. Will help you in your resume, if related to your field. Usually $15per hour.

- Library/Printing/Gym/Other - difficult to get, workload usually easy, apply via handshake or internal referrals work. It will not help in your resume, usually $11.80 per hour

Jobs are available for students but the number of jobs are less due to covid. Be on the lookout for opportunities! All the best!

**Reply by pinazs** [reply]
What would be the right time to start looking for on-campus jobs? @snigi-gupta

**Reply by snigi-gupta** [reply]
You should look for it one sem before. If you want on-campus job in fall 2021 then you should be applying now (spring 2021).
